components/About.js

```diff
@@ -4,7 +4,7 @@ const About = () => (
   <div className="about" id="mission">
     <div className="about__content">
       <div>
-        <Img className="about__logo mb6" src="/static/img/logo.svg" alt="Mozilla Logo" />
+        <Img className="about__logo mb6" src="static/img/logo.svg" alt="Mozilla Logo" />
```

Comment:
`Logo` can be `logo` (or just `Mozilla`)

Reply:
see https://github.com/MozVR/vr-mozilla-org/pull/11/files#diff-fbd6b218b7780f00a345406be15234d1R8 too